tests/suites/test_suite_psa_crypto.function

```diff
@@ -31,6 +31,29 @@
 #define ASSERT_OPERATION_IS_ACTIVE(   operation ) TEST_ASSERT( operation.id != 0 )
 #define ASSERT_OPERATION_IS_INACTIVE( operation ) TEST_ASSERT( operation.id == 0 )
 
+#if defined(PSA_WANT_ALG_JPAKE)
+void ecjpake_operation_setup( psa_pake_operation_t *operation,
+                              psa_pake_cipher_suite_t *cipher_suite,
+                              psa_pake_role_t role,
+                              mbedtls_svc_key_id_t key,
+                              size_t key_available )
+{
+    *operation = psa_pake_operation_init();
```

Comment:
Don't use `*op = init()` to reset an operation: this only works if the operation object happens not to contain pointers (or any other form of resource handles). If the object does contain pointers, this leaks the memory pointed to.

```suggestion
    PSA_ASSERT( psa_pake_abort( *operation ) );
```